alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -39,7 +39,7 @@ impl Drop for Conpty {
 unsafe impl Send for Conpty {}
 
 pub fn new(config: &PtyConfig, window_size: WindowSize) -> Option<Pty> {
-    let mut pty_handle = 0 as HPCON;
+    let mut pty_handle = 0isize;
```

Comment:
Ditto wrt `HPCON` it's present in windows-sys as well.

Reply:
Same, I'll do it this way, it's better.